verisigndns/src/test/java/denominator/verisigndns/VerisignDnsTest.java

```diff
@@ -0,0 +1,224 @@
+package denominator.verisigndns;
+
+import java.util.Iterator;
+
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.Parameterized.Parameter;
+
+import com.squareup.okhttp.mockwebserver.MockResponse;
+
+import denominator.AllProfileResourceRecordSetApi;
+import denominator.DNSApiManager;
+import denominator.Live;
+import denominator.Live.UseTestGraph;
+import denominator.ZoneApi;
+import denominator.common.Util;
+import denominator.model.ResourceRecordSet;
+import denominator.model.Zone;
+
+@RunWith(Live.class)
+@UseTestGraph(VerisignDnsTestGraph.class)
+public class VerisignDnsTest {
```

Comment:
this class seems to repeat work done in the normal live tests. also, it is storing XML, which seems distracting. Should this become a mock test? or should we remove it?